addons/l10n_in_check_gst_status/models/res_partner.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _
+from odoo.addons.iap import jsonrpc
+from odoo.exceptions import UserError, AccessError
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_in_gstin_verified_status = fields.Char(string="GSTIN Verified Status", tracking=True)
+    l10n_in_gstin_verified_date = fields.Date(string="GSTIN Verified Date")
+
+    def get_verified_status(self):
+        url = "https://jva-odoo-iap-apps2-15-0-gst-number-active-check-jva-13181640.dev.odoo.com/iap/l10n_in_reports/1/public/search"
+        user_token = self.env["iap.account"].get("l10n_in_edi")
+        uuid = self.env["ir.config_parameter"].sudo().get_param("database.uuid")
+        for partner in self:
```

Comment:
Are we gonna multiple requests?! every partner!!!! the records needs to be definitely needs to be filtered before hand